src/views/ShoppingList.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+  <div class="todo-page orange lighten-5" id="shoppinglist-page-container">
+    <v-text-field
+      v-model="newItem"
+      @click:append.prevent="addShoppingListItem"
+      @keyup.enter.prevent="addShoppingListItem"
+      class="pa-2 orange lighten-5"
+      outlined
+      label="Add an item"
+      append-icon="mdi-pencil-plus-outline"
+      clearable
+      hide-details
+    >
+    </v-text-field>
+    <v-list
+      v-if="checkShoppingList.length"
+      class="pa-0 orange lighten-5"
+      two-line
+      flat
+    >
+      <!-- Each Todo in Todo list -->
```

Comment:
Great use of previous resources! Don't forget to change the comments if possible though so we don't get confused! haha!

Reply:
This will come in the 1.0 PR ;)

Reply:
Thanks for changing it !